tutorials/usecase/cesna_simulation.py

```diff
@@ -0,0 +1,78 @@
+# bla bla bla
+import numpy as np
+import scipy
```

Comment:
scipy is not installed with decomon. So we need to add in the notebook text that the user needs to install scipy (like we did for pandas in tuto 2). And we need to add it to binder environment also.